go/vt/vtcombo/tablet_map.go

```diff
@@ -762,6 +762,15 @@ func (itmc *internalTabletManagerClient) GetPermissions(ctx context.Context, tab
 	return t.tm.GetPermissions(ctx)
 }
 
+// GetGlobalStatusVars is part of the tmclient.TabletManagerClient interface.
+func (itmc *internalTabletManagerClient) GetGlobalStatusVars(ctx context.Context, tablet *topodatapb.Tablet, statuses []string) (map[string]string, error) {
+	t, ok := tabletMap[tablet.Alias.Uid]
+	if !ok {
+		return nil, fmt.Errorf("tmclient: cannot find tablet %v", tablet.Alias.Uid)
```

Comment:
Nit, but I'd Leo to see us using `topoproto.TabletAliasString(tablet.Alias)` throughout the code base. I'm trying to switch to that over time. It's just much nicer as it maps directly to what users see in `vtctldclient GetTablets`, `show vitess_tablets` etc.